nsqd/http.go

```diff
@@ -464,8 +464,14 @@ func (s *httpServer) doStats(w http.ResponseWriter, req *http.Request, ps httpro
 	formatString, _ := reqParams.Get("format")
 	topicName, _ := reqParams.Get("topic")
 	channelName, _ := reqParams.Get("channel")
+	withProducers, _ := reqParams.Get("with_producers")
 	jsonFormat := formatString == "json"
 
+	var producersStats []ClientStats
+	if withProducers == "1" {
```

Comment:
However, with that, I would argue that the name of the param should be `producers`, not `with_producers`.